src/ardrive.ts

```diff
@@ -77,6 +116,57 @@ export class ArDriveAnonymous extends ArDriveType {
 		const driveEntity = await this.arFsDao.getPublicDrive(driveId);
 		return Promise.resolve(driveEntity);
 	}
+
+	async getPublicFolder(folderId: string): Promise<ArFSPublicFolder> {
+		const folder = await this.arFsDao.getPublicFolder(folderId);
+		return folder;
+	}
+
+	async getChildrenOfPublicFolder(folderId: FolderID): Promise<ArFSPublicFileOrFolderData[]> {
```

Comment:
Perhaps we just name this `listFolder` and provide a docblock explaining what will be returned? Thing to mention might be how deep to expect the listing to go, what revisions of the children to expect getting, etc.

Reply:
Added a JSDoc for that. It will need some rework after we add support for the revisions
9779e1e1cded7e7e476fa754eab82be862e1d0f8